gbpservice/neutron/nsf/config_agent/rpc_cb.py

```diff
@@ -0,0 +1,57 @@
+import importlib
+
+from gbpservice.neutron.nsf.core import periodic_task as core_pt
+from gbpservice.neutron.nsf.config_agent import RestClientOverUnix as rc
+from gbpservice.neutron.nsf.config_agent import loadbalancer as lb
+from gbpservice.neutron.nsf.config_agent import firewall as fw
+from gbpservice.neutron.nsf.config_agent import vpn as vpn
+from gbpservice.neutron.nsf.config_agent import generic as gc
+
+
+CONFIG_AGENT_MODULES = {'loadbalancer': lb,
+                        'firewall': fw,
+                        'vpn': vpn,
+                        'generic': gc}
+
+
+class RpcCallback(core_pt.PeriodicTasks):
+
+    def __init__(self, sc):
+        self._sc = sc
+
+    def handle_event(self, ev):
+        self._sc.poll_event(ev)
+
+    def _method_handler(self, rpc_cb):
+        if rpc_cb['receiver'] == 'orchestrator':
+            mod = CONFIG_AGENT_MODULES['generic']
+            clazz = getattr(mod, 'generic'.title())()
```

Comment:
since CallBack classes from each resource file(Firewall,Vpn etc) is moved, 
removed line no. 28. changed name of method to mod_method